components/brave_ads/core/internal/account/wallet/wallet.h

```diff
@@ -18,6 +18,9 @@ class Wallet final {
  public:
   bool Set(const std::string& payment_id,
            const std::vector<uint8_t>& recovery_seed);
+
+  bool SetFrom(const WalletInfo& wallet);
```

Comment:
Can we please add a comment to state this is a temporary fix until 1.54.x lands which has a more robust solution